fetch.bs

```diff
@@ -5141,8 +5166,11 @@ constructor must run these steps:
  <a for=request>redirect mode</a>,
  <a for=request>integrity metadata</a> is
  <var>request</var>'s
- <a for=request>integrity metadata</a>, and
- <a>keepalive flag</a> is <var>request</var>'s <a>keepalive flag</a>.
+ <a for=request>integrity metadata</a>,
+ <a>keepalive flag</a> is <var>request</var>'s <a>keepalive flag</a>,
+ <a>reload-navigation flag</a> is  <var>request</var>'s <a>reload-navigation flag</a>,
+ <a>history-navigation flag</a> is  <var>request</var>'s <a>history-navigation flag</a>, and
+ <a>submission-navigation flag</a> is  <var>request</var>'s <a>submission-navigation flag</a>.
```

Comment:
@annevk do you think this should become an unordered list? It's becoming a bit tough to read.

Reply:
We could do that, though if we do we should perhaps also clean up the other 2-3 "a new request" places to make them all similar.

Also, if we don't, we should remove the double spaces here.